src/main/java/com/cachinginapp/enterprise/dto/Location.java

```diff
@@ -8,6 +8,8 @@
     private String state;
     @SerializedName("County")
     private String county;
+    @SerializedName("UserID")
```

Comment:
added this for the user

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.